packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js

```diff
@@ -686,6 +686,15 @@ export default class WebRTCMediaEngine {
   triggerNegotiationNeeded() {
     this.trigger(`negotiationneeded`);
   }
+
+  /**
+   * Returns a string when attempting to serialize object
+   * (Applies mostly this is a property of AmpState object)
```

Comment:
wording?

Reply:
Deep. I re-edited that three times. Guess it wasn’t a charm

Reply:
Derp